tools/SwiftSyntax/README.md

```diff
@@ -1,4 +1,68 @@
-# SwiftLanguage
+# SwiftSyntax
 
-This is an in-progress implementation of a Swift API for the
-[libSyntax](https://github.com/apple/swift/tree/master/lib/Syntax) library.
+SwiftSyntax is a set of Swift bindings for the
+[libSyntax](https://github.com/apple/swift/tree/master/lib/Syntax) library. It
+allows for Swift tools to parse, inspect, generate, and transform Swift source
+code.
+
+> Note: SwiftSyntax is still in development, and the API is not guaranteed to
+> be stable. It's subject to change without warning.
+
+## Usage
+
+First, install the latest Swift Master toolchain from swift.org. This will
```

Comment:
After installing the toolchain, we need to switch to it in Xcode. Also, a link will help.